openshift/02-apicast-builder-imagestream.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: apicast
+  labels:
+    app: apicast
+spec:
+  tags:
+  - name: builder
+    from:
+      kind: DockerImage
+      name: quay.io/3scale/apicast:master-builder
```

Comment:
Shuld this use https://docs.openshift.com/container-platform/3.6/architecture/core_concepts/builds_and_image_streams.html#image-stream-mappings-working-periodic ?

Reply:
That could be set as https://docs.openshift.com/container-platform/3.6/rest_api/openshift_v1.html#v1-tagimportpolicy

Reply:
Yes I think it is a good idea.